.changeset/proud-tigers-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sigle/tailwind-style': patch
+---
+
+Fix underline on links showing up outside of article.
```

Comment:
could you update the message of this one and add a changeset for the app please?